cdc/owner.go

```diff
@@ -1626,3 +1633,9 @@ func (o *Owner) startCaptureWatcher(ctx context.Context) {
 		}
 	}()
 }
+
+// reset gcBLT
+func (o *Owner) setGcBLT(gcBLT int64) {
+	o.gcBLT = gcBLT
+	log.Warn("change gc-ttl", zap.Int64("gc-ttl", o.gcBLT))
```

Comment:
`gc-ttl`? type?

Reply:
You are right... I will correct it.  thankssssss!